project_national/fact_sheets_category_5.yml

```diff
@@ -0,0 +1,277 @@
+
+postprocessing:
+  keep_individual_timeseries: false
+  aws:
+    region_name: us-west-2
+    s3:
+      bucket: largeee
+      prefix: test_runs
+    athena:
+      glue_service_role: service-role/AWSGlueServiceRole-default
+      database_name: largeee_test_runs
+      max_crawling_time: 1200
+
+workflow_generator:
+  type: residential_hpxml
+  args:
+    build_existing_model:
+      simulation_control_timestep: 60
+      simulation_control_run_period_begin_month: 1
+      simulation_control_run_period_begin_day_of_month: 1
+      simulation_control_run_period_end_month: 12
+      simulation_control_run_period_end_day_of_month: 31
+      simulation_control_run_period_calendar_year: 2007
+
+    emissions:
+      - scenario_name: LRMER_MidCase_15
+        type: CO2e
+        elec_folder: data/cambium/LRMER_MidCase_15
+
+    utility_bills:
+      - scenario_name: Bills
+
+    simulation_output_report:
+      timeseries_frequency: monthly
+      include_timeseries_total_consumptions: true
+      include_timeseries_fuel_consumptions: true
+      include_timeseries_end_use_consumptions: true
+      include_timeseries_emissions: true
+      include_timeseries_total_loads: false
+
+    reporting_measures:
+      - measure_dir_name: QOIReport
+
+    server_directory_cleanup:
+      retain_in_idf: false
+      retain_schedules_csv: false
+
+### -- Upgrades ---
+# Category 5:  Heat Pumps with Electric Backup
+# To do: costing interpolation from Less for 9.2, 9.5, and 11 HSPF
+upgrades:
+  - upgrade_name: EnergyStar ducted ASHP w/ elec backup  #also provides SEER 15 cooling measure profile
+    # Measure 1
+    options:
+      #ASHP
+      - &ducted_ASHP_EnergyStar_elec_backup
+        option: HVAC Heating Efficiency|ASHP, SEER 16, 9.2 HSPF # same as base option
+        apply_logic: &ducted_ASHP_elec_backup
+          - HVAC Has Ducts|Yes
+          - or: &not_ASHP_greater_than_SEER_15_9_HSPF
+            - not: HVAC Cooling Efficiency|Heat Pump
+            - HVAC Heating Efficiency|ASHP, SEER 10, 6.2 HSPF
+            - HVAC Heating Efficiency|ASHP, SEER 13, 7.7 HSPF
+            - HVAC Heating Efficiency|ASHP, SEER 15, 8.5 HSPF
+        costs: &costs_ashp_ducted_SEER_16_9-2_HSPF
+        # Source: Custom regression by Brennan Less. Ducted heat pump project costs were regressed on nameplate tons and HSPF,
+        # from data described in in Less et al. https://doi.org/10.20357/B7FP4D. 2019$ inflated to 2022$ using 1.12 factor.
+        # Ducted ASHP regression n=317. Implicitly includes electrical upgrades and electric backup.
+          - value: 3907.01 # for HSPF 9
+            multiplier: Fixed (1)
+          - value: 155.17
+            multiplier: Size, Heating System Primary (kBtu/h)
+        lifetime: &lifetime_ducted_ASHP_elec_backup 15
+
+      - &HP_cooling_efficiency
+        option: HVAC Cooling Efficiency|Heat Pump
+        apply_logic:
+          - or:
+            - *ducted_ASHP_elec_backup
+            - *ductless_MSHP_elec_backup
```

Comment:
I think this measure is for ASHP only, so no need to use logic: ductless_MSHP_elec_backup

Reply:
Does this mean I should make separate anchor references for 100_percent conditioned and HP_cooling_efficiency for the ductless measures? The original reason I added ductless to this was so I wouldn't have to make as many reference anchors.

Reply:
I understand, but you'll want to separate references here, again because this package is ASHP applied to ducted only, so this option should not touch the ductless at all. Recall we get average upgrade saving by examining changes in applicable buildings only, changing the ductless here would mean applicability of the package is no longer limited to ducted only. @whiphi92

Reply:
Alternatively, I think you can leave this anchor as is but add package_apply_logic to enforce filtering to ducted only. Will need to be verified with the Upgrade_analyzer.